scripts/boxesserver

```diff
@@ -241,8 +241,9 @@ class BServer:
                 continue
             if len(group._group_actions) == 1 and isinstance(group._group_actions[0], argparse._HelpAction):
                 continue
-            prefix = getattr(group, "prefix", None)
-            result.append('''<h3 id="h-%s" class="open" onclick="showHide(%s)">%s</h3>\n<table role="presentation" id="%s">\n''' % (groupid, groupid, _(group.title), groupid))
+            prefix = getattr(group, "prefix", None) 
```

Comment:
Adds event handler for keyboard enter press and tab index to allow for tab navigation on clickable headings and a more device independent event handler.